src/crimson/os/seastore/random_block_manager/block_rb_manager.h

```diff
@@ -65,7 +65,7 @@ class BlockRBManager final : public RandomBlockManager {
     return device->get_journal_start() + device->get_journal_size();
```

Comment:
does device->get_journal_size() include both journal_head and journal ?

Reply:
Right.